src/services/value-stat-getter.js

```diff
@@ -1,4 +1,6 @@
+import Interface from 'forest-express';
```

Comment:
[MISC] You're only using the `scopeManager` from FE, can't you just destructure it?

Reply:
I did that to all files before, but changed it back.
The reason was that destructuring broke the tests because many tests do `Interface.schema = { mockdata }`

In this case it would work